docs/source/contents/4_draft_mapping.rst

```diff
@@ -1937,32 +1974,32 @@ Topical and name subjects with URIs
 Use Case
 ^^^^^^^^
 
-Remediated collections include subject values with URIs.
+Remediated collections include :code:`subject` values with URIs.
 
 Justification
 ^^^^^^^^^^^^^
 
-In migration, subjects with name and topical values will be treated in the same way. We have decided that the previous
-distinction between name and topical values as subjects is not essential - only the presence of all the values in the
+In migration, :code:`subject`\ s with :code:`name` and :code:`topical` values will be treated in the same way. We have decided that the previous
```

Comment:
it's `topic`, not `topical`.

Reply:
Yes, this was intended as text (an adjective) and not as code. If we are making these code, we'll need to change them to topic.

Reply:
No editing and listening to Keel. :man_shrugging: 

Thanks for catching this!